core/src/main/java/org/elasticsearch/search/aggregations/InternalAggregation.java

```diff
@@ -140,6 +135,11 @@ public String getName() {
      */
     public abstract Type type();
 
+    @Override
+    public String getWriteableName() {
+        return type().stream().toUtf8();
```

Comment:
Yeah, .stream is a bytesarray because the streams class liked it that way.
On Apr 1, 2016 5:30 AM, "Luca Cavanna" <notifications@github.com> wrote:

> In
> core/src/main/java/org/elasticsearch/search/aggregations/InternalAggregation.java
> <https://github.com/elastic/elasticsearch/pull/17463#discussion_r58182387>
> :
>
> > @@ -140,6 +135,11 @@ public String getName() {
> >       */
> >      public abstract Type type();
> >
> > +    @Override
> > +    public String getWriteableName() {
> > +        return type().stream().toUtf8();
>
> is the name the bytes array itself?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/elasticsearch/pull/17463/files/df01decb76a21d3731f65426cebe2f5d12dc5185#r58182387>
>